src/twisted/web/test/test_http2.py

```diff
@@ -2938,6 +2937,8 @@ async def test_realRequest(self) -> None:
         """
         Twisted's HTTP/2 server can talk to a third-party HTTP/2 client.
         """
+        from twisted.test.test_sslverify import certificatesForAuthorityAndServer
```

Comment:
Circular import? It feels like this belongs in `twisted.internet.testing` or `twisted.test.ssl_helpers`.

Reply:
I don't want to make it public, and I'm not confident in its factoring, so I'd rather just keep this one janky import rather than do a big project of refactoring it :)